contentreview-impl/tool/src/java/org/sakaiproject/contentreview/servlet/GradingCallbackServlet.java

```diff
@@ -196,6 +208,21 @@ protected void doPostXml(HttpServletRequest request, HttpServletResponse respons
         return;
     }
 	
+	public Map<String,Object> obtainGlobalTurnitinLTITool(String turnitinSite){
```

Comment:
Wouldn't it be better to expose the LTI configuration from the service and just have the grading callback servlet get the config from there, rather than duplicating the code?